lib/utils/openBrowser.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+
+const opn = require('open');
+const chalk = require('chalk');
+const isDockerContainer = require('is-docker');
+const BbPromise = require('bluebird');
+
+function displayManualOpenMessage(url, err) {
+  // https://github.com/sindresorhus/log-symbols
+  console.log('---------------------------');
+  const errMsg = err ? `\nError: ${err.message}` : '';
+  const msg = `Unable to open browser automatically${errMsg}`;
+  console.log(`🙈  ${chalk.red(msg)}`);
```

Comment:
While cute I'm unsure if we want to go the "emotification of all the CLIs" route in an enterprise-facing project.

Reply:
I copy and pasted this out of `platform-sdk` 😬

Reply:
(as to why i copied, it.. didn't want to add sdk as a direct dep, which is the only way to guarantee it's availability with things like `yarn`, and now also the fact it requires newer versions of node)

Reply:
> now also the fact it requires newer versions of node

But SDK Is dependency of a plugin.. so if SDK was updated to not work in v6, then through Plugin  it'll also break SLS (?)